style.css

```diff
@@ -351,3 +368,172 @@ textarea::placeholder {
 input::placeholder {
   color: #979493;
 }
+
+@media screen and (min-width: 768px) {
+  header {
+    padding: 24px 150px;
+    height: 32px;
+  }
+
+  a.nav-menu {
+    display: none;
+  }
+
```

Comment:
- [ ] Your footer has excess height. Please check that.
![image](https://user-images.githubusercontent.com/84156714/193418726-50d9ba00-91c3-4740-ac00-d350f125e239.png)